frontend/src/App.js

```diff
@@ -1,26 +1,29 @@
 import React, { Component } from 'react';
-import { Switch, Route } from 'react-router-dom';
+import { Switch, Route, withRouter } from 'react-router-dom';
 import Homepage from './containers/Homepage/Homepage.js';
 import LoginPage from './containers/LoginPage/LoginPage.js';
 import SignUp from './components/SignUp/SignUp.js';
 import UsersAll from './components/AdminControls/UsersAll.js';
 import User from './components/AdminControls/User.js';
-import history from './history.js';
+// import history from './history.js';
```

Comment:
I tend to think deleting unused code is preferable to commenting it out. We can always go find it in the git history if we need it, and until then it's just clutter that makes the code harder to read.